### ISSUE TITLE
Toolbar and Panel: Screen Reader doesn't announce title of panel and toolbar header.

### ISSUE THREAD

**Issue by AbhivyaktiYadav** [SAP/ui5-webcomponents-react#6991]
### Describe the bug

<img width="686" alt="Image" src="https://github.com/user-attachments/assets/5f46fd6f-bf32-43b6-8839-2494dd5a6eaa" /> 
For example, in this screenshot from the StackBlitz link, the panel title "All Categories" is not announced by screen reader.

### Isolated Example

https://stackblitz.com/edit/vitejs-vite-vhu4gpem?file=src%2FPanelComponent.tsx,src%2FToolbarComponent.tsx,src%2FApp.tsx

### Reproduction steps

1.
2.
3.
...

### UI5 Web Components for React Version

~1.29.0

### UI5 Web Components Version

~1.24.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @AbhivyaktiYadav 

you're example doesn't render as you forgot to add the `ThemeProvider` component (see error in console). --> [fixed example](https://stackblitz.com/edit/vitejs-vite-rt3uk7yq?file=src%2Fmain.tsx,src%2FApp.tsx)

Also, please provide the following information:

- Since you're using a tilde to define the version, I'm assuming you're using the current latest v1 version (ui5wc: 1.24.16, ui5wcr: 1.29.17). Is that correct? 
- You mention the Toolbar in the issue title, but don't describe how it's connected to the problem, neither in the issue description or the example. Is the Toolbar relevant or not?
- What screen reader are you using (including version)?

In general, please try following our [Contribution Guidelines](https://github.com/SAP/ui5-webcomponents-react/blob/main/CONTRIBUTING.md).
